src/components/testCaseList/TestCaseList.tsx

```diff
@@ -3,24 +3,35 @@ import tw from "twin.macro";
 import "styled-components/macro";
 import useTestCaseServiceApi from "../../api/useTestCaseServiceApi";
 import TestCase from "../../models/TestCase";
-import { useNavigate, useParams } from "react-router-dom";
+import { useParams } from "react-router-dom";
+import { Button } from "@madie/madie-components";
+import TestCaseComponent from "./TestCase";
+import TestCasePopulationList from "./TestCasePopulationList";
+
 const TH = tw.th`p-3 border-b text-left text-sm font-bold uppercase`;
-const TD = tw.td`p-3 whitespace-nowrap text-sm font-medium text-gray-900`;
-const Button = tw.button`text-blue-600 hover:text-blue-900`;
 const ErrorAlert = tw.div`bg-red-100 text-red-700 rounded-lg m-1 p-3`;
 
 const TestCaseList = () => {
-  const navigate = useNavigate();
   const [testCases, setTestCases] = useState<TestCase[]>();
   const [error, setError] = useState("");
   const { measureId } = useParams<{ measureId: string }>();
   const testCaseService = useTestCaseServiceApi();
 
+  const [isExecuteButtonClicked, setisExecuteButtonClicked] = useState(false);
+  const [activeItem, setActiveItem] = React.useState(null);
+  //MAT-3911: only for mock up purpose
+  let lastTestCase = null;
+
   useEffect(() => {
     if (!testCases) {
       testCaseService
         .getTestCasesByMeasureId(measureId)
         .then((testCaseList: TestCase[]) => {
+          //MAT-3911: only for mock up purpose
+          const listItems = testCaseList.map((testCase) => {
+            testCase.executionStatus = "NA";
+            lastTestCase = testCase;
```

Comment:
Is `lastTestCase` used anywhere else? If not, can it be removed?

Reply:
Can you pls try removing it and see if it breaks(also linter is suggesting to not update variable directly in useEffect).

Reply:
@adongare It's removed.